snow/consensus/snowball/nnary_slush.go

```diff
@@ -11,6 +11,12 @@ import (
 
 var _ NnarySlush = (*nnarySlush)(nil)
 
+func newNnarySlush(choice ids.ID) nnarySlush {
```

Comment:
I feel like we dont need this function and we can just use the struct directly

Reply:
I feel like I kinda want to keep them just for consistency... In general all the `Nnary` stuff could be moved to a test file...